### DISCUSSION TITLE
[feature request] Double click driver in (replay) live timing window to open their onboard stream

### DISCUSSION THREAD

**Discussion by mozjag** [f1multiviewer/issue-tracker#570] [Feature requests]
Just a convenience. I obviously can alt-tab back to the main window, find and open the driver's stream, and then minimize the main window (again), but double-clicking their name in the live timing window would make it more convenient to e.g. find out why they had a slow sector / lap. (Especially once #310 is implemented.)

**Comment by jeppeolesen** [comment]
Marking this as a duplicate of #369
